### ISSUE TITLE
Consider making the visualisations accessible to users who are blind, if possible

### ISSUE THREAD

**Issue by u451f** [u451f/almanac.article19.org#28]
Plotly charts are not accessible, i.e. screen readers cannot read the contents of what is visible there. On top if that, we make heavy use of Javascript to load the data. 

I need to look into the WCAG2.2 accessibility guideline as to what it suggests we offer to blind users instead. Currently I'm thinking of linking the CSV data on Github - that should be readable by screen readers. (done in 2e68a5b) This is a rather technical solution.

However, this solution likely does not give blind users a sense of what the data reveals. So we could think about - at least for the landing page - having short textual descriptions of the findings. I would integrate them as hidden text - only revealed to screen readers (and potentially search engines). This would allow blind users to get a sense of what the research has brought about.
